.travis.yml

```diff
@@ -0,0 +1,111 @@
+#-- copyright
+# OpenProject is a project management system.
+# Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See doc/COPYRIGHT.rdoc for more details.
+#++
+
+# Travis configuration based on the respective OpenProject core configuration.
+# Everything save for the matrix section and additional `before_install`
+# instructions is copied and pasted from the core.
+
+language: ruby
+
+rvm:
+  - 2.2.3
+
+sudo: false
+
+cache:
+  - bundler: true
+  - directories:
+    - frontend/node_modules
+    - frontend/bower_components
+
+bundler_args: --without development production
+
+branches:
+  only:
+    - master
+    - dev
+    - /^(stable|release)\/.*$/
+
+env:
+  global:
+    - CI=true
+    - RAILS_ENV=test
+    - COVERAGE=true
+
+  matrix:
+    - "TEST_SUITE=plugins:spec      DB=mysql"
+    - "TEST_SUITE=plugins:cucumber  DB=mysql"
+
+before_install:
+  # Custom plugin instructions follow.
+
+  # Move the plugin into a subfolder. The plugin-provided Gemfile.plugins
+  # must refer to this folder.
+  - mkdir -p plugins/this
+  - echo `ls -a | tail -n+3 | grep -v plugins` plugins/this/ | xargs mv
+
+  # Get OpenProject.
+  # Doing the fetch detour as you cannot clone into the current directory.
+  - git init
+  - git remote add openproject https://github.com/opf/openproject.git
+  - git fetch --depth=1 openproject
+  - git checkout openproject/dev
```

Comment:
Yeah ideally this would be the same branch against which the PR is created. No idea how we could do that, though. I mean yes, we can use a variable. But how do we get the branch (i.e. `stable/5` in this case, `dev` in others) in there automatically?